plopfile.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * Defines a plop (https://plopjs.com/) generator to scaffold out new components
+ *
+ * Run with `yarn run plop` or `npm run plop`
+ */
+export default function (plop) {
+  plop.setGenerator('component', {
+    description: 'Add a new component',
+    prompts: [
+      {
+        type: 'input',
+        name: 'name',
+        message: 'component name (e.g. Panel)',
+      },
+      {
+        type: 'input',
+        name: 'group',
+        message: 'component group directory (e.g. layout)',
+      },
+      {
+        type: 'list',
+        name: 'category',
+        message: 'component category',
+        choices: ['presentational', 'composite', 'simple'],
+        default: 'presentational',
+      },
+      {
+        type: 'confirm',
+        name: 'addPatternLibraryPage',
+        message: 'Add a pattern-library documentation page for this component?',
+        default: false,
+      },
+    ],
+    actions: function (data) {
```

Comment:
Unless there is a reason this needs to be a classic function, I'd change it to an arrow function for consistency.